app/src/main/res/layout/list_bp_history_item.xml

```diff
@@ -70,12 +70,12 @@
     android:layout_marginStart="@dimen/spacing_8"
     android:text="@string/patientsummary_edit"
     app:layout_constraintBottom_toTopOf="@id/bottomGuideline"
-    app:layout_constraintEnd_toStartOf="@id/timeDateTextView"
+    app:layout_constraintEnd_toStartOf="@id/dateTimeTextView"
     app:layout_constraintStart_toEndOf="@id/readingsTextView"
     app:layout_constraintTop_toBottomOf="@id/topGuideline" />
 
   <TextView
-    android:id="@+id/timeDateTextView"
+    android:id="@+id/dateTimeTextView"
```

Comment:
It is not necessary to change the names of view IDs to match how we're displaying a certain value. The only time we will need to change the IDs is if thing we are displaying here itself has changed (say, from a timestamp to something else).